modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -28,3 +30,14 @@ function stanford_profile_styles_page_attachments(array &$attachments) {
   }
 
 }
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function stanford_profile_styles_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+
+  if ($context['widget']->getPluginId() == 'paragraphs') {
+    // Add the admin CSS library.
+    $element['#attached']['library'][] = 'stanford_profile_styles/admin';
```

Comment:
I think there is a disconnect in your naming of this library and where/how it is attached. If you are creating an `admin` library I would have expected it to be attached everywhere the admin theme is used and not just when a paragraph is present. We've likely got more changes coming to the admin theme so attaching to all pages in the admin theme should be ok.

Reply:
👍

Reply:
You'll have to change the hook as well to target all admin pages. Perhaps: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/function/hook_page_attachments/8.8.x

My previous comment was to either change the name of the library/css file so that it is specific to the field and conveys that intent, or, to leave it generically named as admin and attach it to all admin pages. With the field_widget_form_alter hook you are still only targeting a small section of the admin ui.

Reply:
Yes i would prefer to split the css into different libraries. even if its one global admin style sheet and one field style sheet for all fields.

Reply:
Hi @sherakama and @pookmish,
Thank you for the explanation.

I went ahead and created an "admin" directory and modeled it after the [admin directory on Stanford Basic](https://github.com/SU-SWS/stanford_basic/tree/8.x-4.x/src/scss/admin).
I think that should make it simpler to add new files and it will make it easier to indicate the files for the authoring environment. 

Please let me know if this will work.